torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2038,6 +2038,10 @@ def _maybe_evaluate_static(self, expr: "sympy.Expr", *, unbacked_only: bool = Fa
             new_shape_env[k] = s + offset
             new_range_env[s] = ValueRangeAnalysis.sub(vr, offset)
 
+        try:
+            sympy.simplify(expr.xreplace(new_shape_env))
+        except Exception:
```

Comment:
Suggest to use specific `Exception` subclass you expect to catch and add related note.